_content/articles/nun-of-milan_dhammadina.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "The nun of Milan: A Gandharan bhikṣuṇī figurine in the Civico Museo Archeologico"
+authors:
+  - dhammadina
+  - Giovanni Verardi
+external_url: "https://www.academia.edu/114584851/_with_Giovanni_Verardi_The_nun_of_Milan_a_Gandharan_bhik%E1%B9%A3u%E1%B9%87%C4%AB_figurine_in_the_Civico_Museo_Archeologico"
```

Comment:
good source url, let's use archive.org for the external_url: https://archive.org/download/aririab-vol-xxvii/Gandharan%20bhik_u%E1%B9%87%C4%AB%20figurine%20in%20the%20Civico%20Museo%20Archeologico.pdf

Reply:
Still not correct